.github/workflows/android.yml

```diff
@@ -0,0 +1,58 @@
+name: Testing polyPod in Android
+on:
+    push:
+    pull_request:
+        types: [assigned, opened, ready_for_review]
+
+jobs:
+    test:
+        name: Testing polyPod app
+        runs-on: macos-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 16.8
+            - name: Checks paths to rebuild Android
+              uses: dorny/paths-filter@b2feaf19c27470162a626bd6fa8438ae5b263721 #v2.10.2
+              id: changes
+              with:
+                  filters: |
+                      android:
+                      - 'android/**'
+                      - 'core/api/**'
+                      - 'core/communication/**'
+                      - 'core/utils/silly-i18n/**'
+                      - 'features/**'
+                      - 'podjs/**'
+                      js:
+                      - '**/*.js'
+                      - '**/*.jsx'
+                      - '**/*.ts'
+                      - '**/*.tsx'
+                      - '**/*.?js'
+                      - '.editorconfig'
+                      - '.eslintrc.cjs'
+                      coreAndFeatures:
+                      - 'core/**'
+                      - 'features/**'
+                      - 'podjs/**'
+                      - 'build/**'
+            - name: Install global modules
```

Comment:
If you're not testing, you don't need this either. I would strongly suggest you keep activities to a minimum due to the very unstable performance of the macos runner.